sale_order_invoicing_finished_task/views/project_view.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="task_type_edit" model="ir.ui.view">
+    <field name="model">project.task.type</field>
+    <field name="inherit_id" ref="project.task_type_edit"/>
+    <field name="arch" type="xml">
+        <field name="fold" position="after">
+            <field name="invoiceable"/>
+        </field>
+    </field>
+</record>
+
+<record id="edit_project_task_track" model="ir.ui.view">
+    <field name="name">project.task.form.track</field>
+    <field name="model">project.task</field>
+    <field name="inherit_id" ref="project.view_task_form2"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='planned_hours']" position="after">
```

Comment:
Please, apply this PR #825 in this migration.

Reply:
Waiting for that PR to be approved to merge it here

Reply:
You can apply it, @Tardo

Reply:
Done

Reply:
Thank you)